app/concepts/todo/create.rb

```diff
@@ -1,50 +1,38 @@
 class Todo::Create < Trailblazer::Operation
+  include Model
+  model Todo, :create
+
+  contract do
+    property :title, validates: { presence: true }
+  end
 
-  # Logic for creating a TODO
-  #     if `todo_list` is specified
-  #         create the `todo`
-  #         associated with `todo_list`
-  #     else
-  #         create the `todo`
-  #         create a `user`
-  #         create a default `todo_list`
-  #         associate `todo_list` with `user`
-  #         associate `todo` with `todo_list`
   def process(params)
-    # make sure title is not blank
-    return invalid! if params[:todo][:title].blank?
+    validate(params[:todo]) do
```

Comment:
I was actually thinking if I'm supposed to catch it, but went with the previous behavior that returns invalid!
`return invalid! if params[:todo][:title].blank?`